assets/js/pages/ClusterDetails/HanaClusterDetails.jsx

```diff
@@ -150,15 +153,27 @@ function HanaClusterDetails({
               },
               {
                 title: 'SID',
-                content: enrichedSapSystem,
-                render: (content) => (
-                  <SapSystemLink
-                    sapSystemId={content?.id}
-                    systemType="databases"
-                  >
-                    {content?.sid}
-                  </SapSystemLink>
-                ),
+                content: enrichedSapSystems,
+                render: (content) =>
+                  content.every(
+                    (sapSystem) => sapSystem.id && sapSystem.sid
+                  ) ? (
+                    <div>
```

Comment:
We don't need this sort of protection here.
Just using what you have withing the `div` should be enough. Use the `sid` as key, they cannot be repeated.
The `SapSystemLink` component will take care about the scenario where the `id` does not exist, it will go as undefined. Don't worry about that. You'll see in storybook.